code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -301,8 +301,8 @@
 
 
 /obj/item/weapon/storage/secure/briefcase/money
-	name = "suspicious briefcase"
-	desc = "An ominous briefcase that has the unmistakeable smell of old, stale cigarette smoke, and gives those who look at it a bad feeling."
+	name = "secure briefcase"
+	desc = "A large briefcase with a digital locking system."
```

Comment:
If these names are default ones used in secure briefcases you can delete them, they're inherited from parent object anyway.